aquarius/events/decryptor.py

```diff
@@ -14,7 +13,8 @@
 
 def decrypt_ddo(w3, provider_url, contract_address, chain_id, txid, hash):
```

Comment:
@mariacarmina as part of https://github.com/oceanprotocol/provider/pull/384 I've added dataNftAddress as required. Please send it as part of the payload.